tokens/global/typography/lineHeights.js

```diff
@@ -0,0 +1,53 @@
+import typography from './core.js';
+import sizes from './fontSizes.js';
+
+const base = typography.base;
+const fontSizes = sizes.fontSizes;
+
+/*
+  Get fontSize value, divided away from the base font scaling
+  LineHeight has no units, its relative.
+  Find out how many baseline octaves are required to fit the fontSize. Add one octave for a bit more spacing
+  Return numBaseline as a percentage for fontSizeValue
+*/
```

Comment:
I love this!!! Could we switch to JSDoc format?
```suggestion
/**
 * Calculates the line height as a percentage of the font size value.
 * Get fontSize value, divided away from the base font scaling
 * LineHeight has no units, its relative
 * Find out how many baseline octaves are required to fit the fontSize. Add one octave for a bit more spacing
 * @param {string} fontSize - The font size value as a string.
 * @returns {number} The calculated line height as a percentage of the font size value.
 */
```

Also just trying to come up with a better description for the octave computation, but I can't think of a better one ATM